### ISSUE TITLE
Top Talkers missing FQDN toggle feature

### ISSUE THREAD

**Issue by tofazzz** [opnsense/core#5536]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Is your feature request related to a problem? Please describe.**
The FQDN toggle is missing from the Top Talkers tab under the traffic reporting

A clear and concise description of what the problem is including your motivation for the request,
i.e. "For the purpose of [...] I am missing a solution that will [...]."
The FQDN toggle (DNS lookup) in "Top Talkers" section is not there anymore and it was a useful feature for troubleshooting. It would be possible to have it back in?

**Describe the solution you like**
FQDN toggle back in "Top Talkers" section

(e.g. I would like an input field in the `/ui/firewall/alias` which would add .... to ....)

**Describe alternatives you considered**

A clear and concise description of any alternative solutions or features you considered.

**Additional context**

Add any other context or screenshots about the feature request here or links to relevant forum thread or similar

**Comment by AdSchellevis** [comment]
The old code was removed quite some time ago (https://github.com/opnsense/core/commit/d7279fba604983e574e93e3dc46f40e0b90511b7, https://docs.opnsense.org/releases/CE_20.7.html#december-08-2020), a similar option as in Reporting/Insight might be a nice to have, but not very high on our priority list.

**Comment by AdSchellevis** [comment]
https://github.com/opnsense/core/commit/92352ec4fa0ca1dc76dd3098fc35a4aa64f1f2cf should add the reverse fqdn's for the addresses when available. It's a bit of an experiment, but if you do want to try it out, the following command should apply on the latest OPNsense version:

```
opnsense-patch 92352ec
```

**Comment by tofazzz** [comment]
Thank you for the temp patch!

So the idea is to have it on all the time? Why not use a toggle to let the user enable/disable it?

**Comment by AdSchellevis** [comment]
since we resolve in parallel, the assumption is that the additional time needed to handle the request is reasonable, in which case it probably isn't worth the effort of adding more gui glue (both address and name are presented now).
